publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dto/Contributor.java

```diff
@@ -0,0 +1,114 @@
+package no.unit.nva.publication.doi.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.Objects;
+import no.unit.nva.publication.doi.dynamodb.dao.Identity;
+import nva.commons.utils.JacocoGenerated;
+
+public class Contributor {
```

Comment:
Fields for the DTO need to be agreed on. What do we need when updating the DOI @brinxmat ? 

I guess we reserve the `id` for upcoming id from masterdata?
Then we currently use `arpId`, do we want `orcId` to be included in the DOI metadata?